collectors/proc.plugin/proc_stat.c

```diff
@@ -426,8 +426,8 @@ static int read_cpuidle_states(char *cpuidle_name_filename , char *cpuidle_time_
                 return 1;
             }
 
-            name_buf[r - 1] = '\0'; // erase extra character
-            cc->cpuidle_state[state].name = strdupz(name_buf);
+            name_buf[r-1] = '\0';
```

Comment:
Maybe keep `r - 1` as it was before instead of changing it to `r-1` for code style consistency with the rest of the file. You could also keep the comment and make this a 1 line commit instead of a 2 lines commit.

Reply:
Done